Fortinet/fortimail/tests/smtp_event_to_user_unknown.json

```diff
@@ -0,0 +1,48 @@
+{
+  "input": {
+    "message": "time=10:50:36.931 device_id=FEVM020000087260 log_id=0003008733 type=event subtype=smtp pri=information  user=mail ui=mail action=NONE status=N/A session_id=\"1548fVq5008733-1548fVq5008733\" msg=\"to=postmaster, delay=00:00:50, mailer=local, pri=58900, dsn=5.1.1, stat=User unknown(Reason from remote:550 5.1.1 User unknown)\"",
+    "event": {
+      "created": "2021-06-04T01:23:08.541077Z",
+      "id": "12345"
+    },
+    "log": {
+      "hostname": "00000"
+    }
+  },
+  "expected": {
+    "message": "time=10:50:36.931 device_id=FEVM020000087260 log_id=0003008733 type=event subtype=smtp pri=information  user=mail ui=mail action=NONE status=N/A session_id=\"1548fVq5008733-1548fVq5008733\" msg=\"to=postmaster, delay=00:00:50, mailer=local, pri=58900, dsn=5.1.1, stat=User unknown(Reason from remote:550 5.1.1 User unknown)\"",
+    "event": {
+      "action": "NONE",
```

Comment:
I think that `None` should not appear ?

Reply:
I don't know if this value can be useful for detection rules :thinking: